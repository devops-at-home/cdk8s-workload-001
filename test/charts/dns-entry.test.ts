import { App } from 'cdk8s';
import { DnsEntry } from '../../src/charts/dns-entry.chart';

const chart = 'dns-entry';

describe(chart, () => {
    test('Snapshot test', () => {
        const app = new App();

        const manifest = new DnsEntry(app, chart, {
            namespace: 'default',
        });

        expect(manifest.toJson()).toMatchSnapshot();
    });
});
