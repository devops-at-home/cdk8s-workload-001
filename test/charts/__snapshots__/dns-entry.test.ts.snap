// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dns-entry Snapshot test 1`] = `
[
  {
    "apiVersion": "externaldns.k8s.io/v1alpha1",
    "kind": "DNSEndpoint",
    "metadata": {
      "name": "dns-entry-dnsendpoint-c843a586",
      "namespace": "default",
    },
    "spec": {
      "endpoints": [
        {
          "dnsName": "test.tld",
          "recordType": "A",
          "targets": [
            "127.0.0.1",
          ],
        },
      ],
    },
  },
]
`;
